README.md

```diff
@@ -248,7 +254,8 @@ If both `WARM_IP_TARGET` and `MINIMUM_IP_TARGET` are set, `ipamd` will attempt t
 This environment variable overrides `WARM_ENI_TARGET` behavior. For a detailed explanation, see
 [`WARM_ENI_TARGET`, `WARM_IP_TARGET` and `MINIMUM_IP_TARGET`](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/eni-and-ip-target.md).
 
-
+`ENABLE_PREFIX_DELGATION` set to `true` and this environment variable overrides `WARM_PREFIX_TARGET` behavoir. For a detailed explanation, see
```

Comment:
typo for behavior - same as above

Reply:
Formatting seems off here?

Reply:
It seems fine in the editor, will run make format again.